src/components/BottomSheet/BottomSheet.js

```diff
@@ -16,7 +16,7 @@ const BottomSheet = React.forwardRef((props, ref) => {
       dragFromTopOnly={true}
       closeOnPressMask={props.closeOnPressMask}
       height={props.height ? props.height : 355}
-      keyboardAvoidingViewEnabled={false}
+      keyboardAvoidingViewEnabled={keyboardAvoidingViewEnabled}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is good as it allows the `keyboardAvoidingViewEnabled` property to be configurable. However, it's important to ensure that all usages of the `BottomSheet` component have been updated to pass this new prop where necessary. If not, it could lead to unexpected behavior in some parts of the application where the keyboard avoiding view is expected but not enabled.


<!-- This is an auto-generated comment by OSS CodeRabbit -->